django_datadog_logger/formatters/datadog.py

```diff
@@ -70,7 +72,7 @@ def get_wsgi_request_user(wsgi_request):
 
 
 class DataDogJSONFormatter(json_log_formatter.JSONFormatter):
-    def json_record(self, message, extra, record):
+    def json_record(self, message: str, extra: dict, record: LogRecord) -> dict:
```

Comment:
`dict` can be used only in Python 3.9 and up. If we strive for compatibility with lower versions (is 3.7 a reasonable lower bound?) then we need to use `typing.Dict`.

Reply:
Ah yes, good point. Thanks